test/props.test.js

```diff
@@ -0,0 +1,78 @@
+class User extends React.Component {
+  render () {
+    return (
+      <span>User {this.props.index}</span>
+    )
+  }
+}
+
+User.propTypes = {
+  index: React.PropTypes.number.isRequired
+}
+
+class Fixture extends React.Component {
+  render () {
+    return (
+      <div>
+        <ul>
+          <li><User index={1} objectProp={{foo: 'bar'}} /></li>
+          <li><User index={2} /></li>
+        </ul>
+      </div>
+    )
```

Comment:
Unfortunately that doesn't seem to work, I think it might be necessary to retrieve a component from within the wrapper to run the assertions against?

```
AssertionError: expected <Fixture /> to have props [ 'href', 'style' ] but props were []

HTML:
<a data-reactroot="" href="#" style="display: inline-block;">a link</a>
```

If it's not possible to simplify the `expect` call I think there wouldn't be much benefit to changing the fixture structure, but I'm happy to make the change so let me know if I've missed something.